src/event_handler.py

```diff
@@ -21,19 +22,25 @@ def store_queued_events(_, __):
     logger.setLevel(logging.INFO)
 
     if 'ENCRYPTION_KEY' in os.environ:
-      encrypted_decryption_key = os.environ['ENCRYPTION_KEY']
-      logger.info('Got decryption key from environment variable')
+        encrypted_decryption_key = os.environ['ENCRYPTION_KEY']
+        logger.info('Got decryption key from environment variable')
     else:
-      encrypted_decryption_key = fetch_decryption_key()
-      logger.info('Got decryption key from S3')
+        encrypted_decryption_key = fetch_decryption_key()
+        logger.info('Got decryption key from S3')
     decryption_key = decrypt(encrypted_decryption_key)
     logger.info('Decrypted key successfully')
 
+    dsn = os.environ['DB_CONNECTION_STRING']
```

Comment:
Deep Space Nine?

Reply:
Maybe the envar should be called `DATA_SOURCE_NAME`?

Reply:
Maybe a future change ;-)